src/test/java/uk/gov/hmcts/reform/civil/service/docmosis/dq/DirectionsQuestionnaireGeneratorTest.java

```diff
@@ -67,8 +67,9 @@
 
     private static final String BEARER_TOKEN = "Bearer Token";
     private static final String REFERENCE_NUMBER = "000DC001";
+    private static final String USER_PREFIX = "defendant";
```

Comment:
We should also have a test for `claimant` prefix

Reply:
Added in 2229727